README.md

```diff
@@ -25,13 +25,13 @@ We started to create a phar file, starting from the version 0.7.0
 release. It is available at the URLs like:
 
 ```
-https://github.com/satooshi/php-coveralls/releases/download/v0.7.1/coveralls.phar
+https://github.com/satooshi/php-coveralls/releases/download/v1.0.0/coveralls.phar
```

Comment:
This change is compatible with 1.0 line and should be done for 1.0 line.
It's not a feature, it's not a major BC breaking change, so 1.0.x.
(BTW, this will decreace non-needed difference between 1.0 and 1.1 and make merges easier)